### PR TITLE
Support desc keyword

### PR DESCRIPTION
**Pull request by darionyaphet**

### CODE REVIEW

File: src/parser/parser.yy [hunk 1]
```diff
@@ -596,12 +596,18 @@ describe_tag_sentence
     : KW_DESCRIBE KW_TAG name_label {
         $$ = new DescribeTagSentence($3);
     }
+    | KW_DESC KW_TAG LABEL {
```
Comment (dutor) [inline src/parser/parser.yy]: `name_label`

File: src/parser/parser.yy [hunk 2]
```diff
@@ -596,12 +596,18 @@ describe_tag_sentence
...
+        $$ = new DescribeTagSentence($3);
+    }
     ;
 
 describe_edge_sentence
     : KW_DESCRIBE KW_EDGE name_label {
         $$ = new DescribeEdgeSentence($3);
     }
+    | KW_DESC KW_EDGE LABEL {
```
Comment (dutor) [inline src/parser/parser.yy]: ditto.

File: src/parser/parser.yy [hunk 3]
```diff
@@ -941,6 +947,9 @@ describe_space_sentence
     : KW_DESCRIBE KW_SPACE LABEL {
         $$ = new DescribeSpaceSentence($3);
     }
+    | KW_DESC KW_SPACE LABEL {
```
Comment (dutor) [inline src/parser/parser.yy]: ditto.

File: src/parser/parser.yy (hunk 2 above)
Comment (darionyaphet) [reply src/parser/parser.yy]: fixed ~

**Comment by steppenwolfyuetong** [comment]
This modification is not perfect. I think it is more efficient to use synonyms in` scanner.lex`  files.

**Comment by dutor** [comment]
> This modification is not perfect. I think it is more efficient to use synonyms in` scanner.lex` files.

Good point, but this is a special one. Because `DESC` is not just a synonym to `DESCRIBE`, but also a standalone keyword in the comming `ORDERED BY` statement.

I haven't digged up into the impl of MySQL or other RDBMS. If you have some other smarter ideas, please enlighten me.

**Comment by steppenwolfyuetong** [comment]
> Good point, but this is a special one. Because `DESC` is not just a synonym to `DESCRIBE`, but also a standalone keyword in the comming `ORDERED BY` statement

@dutor,    👍，Yeah，`DESC` also uses in  `ORDER BY` statement.  But here `DESC` is equivalent to `DESCRIBE`.
 
so I think  it could be better using as follow:
```
decribe_command:
       KW_DESC
    |  KW_DESCRIBE
    ;

```
use decribe_command

**Comment by darionyaphet** [comment]
Using `synonym` to handle `KW_DESC` and `KW_DESCRIBE` seems not a good idea.

`DESC` is another keyword.  They are not equivalent.

How to deal with `ORDER BY  DESCRIBE` ?

**Comment by darionyaphet** [comment]
In test case `CREATE SPACE default_space` have modify replica_factor from `3` to `1`,  fixed

**Comment by darionyaphet** [comment]
jenkins go

**Pull request merged**